### PR TITLE
Add new devices

### PR DESCRIPTION
**Pull request by mangerlahn**
- iPod Touch 7G
- iPad Pro 3rd-gen (11 + 12.9 inch)
- iPad Mini 5
- iPad Air 3

Source is mostly https://gist.github.com/adamawolf/3048717. I verified all of these using Mactracker and everymac.com

### CODE REVIEW

File: UIDeviceIdentifier/UIDeviceHardware.m [hunk 1]
```diff
@@ -105,6 +106,18 @@ + (NSString *) platformString
             @"iPad7,4": @"iPad Pro 10.5-inch (WiFi/Cellular)",
             @"iPad7,5": @"iPad 6 (WiFi)",
             @"iPad7,6": @"iPad 6 (WiFi/Cellular)",
+            @"iPad8,1": @"iPad Pro 11-inch 3rd-gen (WiFi)",
+            @"iPad8,2": @"iPad Pro 11-inch 3rd-gen (WiFi)",
```
Comment (mangerlahn) [inline UIDeviceIdentifier/UIDeviceHardware.m]: Every second identifier is for the 1TB version. I did not include this detail in the platform string.

Comment (squarefrog) [reply UIDeviceIdentifier/UIDeviceHardware.m]: Thats fine 👍

**Comment by squarefrog** [comment]
This is great, thank you so much!

**Comment by mangerlahn** [comment]
That was fast 😅. Happy to help!

**Pull request merged**